app/src/common/utils/attributeUtils.js

```diff
@@ -26,8 +26,12 @@ export const formatAttribute = ({ key, value } = {}, divider = ':') => {
   return getAttributeValue({ key, value }) || '';
 };
 
+export const formatAttribute = (attribute) => {
```

Comment:
And the name for this function does not look consistent.
We have `formatAttributeWithSpacedDivider`. May be better to rename it to `formatAttributeWithDefaultDivider`?

Reply:
Usually when we use smth without specifiers it means that we expect the default behavior.
So here we have `formatAttribute` - the default way to format attributes (and the frequently used) and the `formatAttributeWithSpacedDivider` which provides non-default behavior for some specific use cases.
Another reason why I decided to leave the default naming as is - to not change the API for plugins where this utility used as well.

Reply:
I got it!